examples/misc_exporter_gltf.html

```diff
@@ -451,6 +454,37 @@
 
 				window.addEventListener( 'resize', onWindowResize );
 
+				// ---------------------------------------------------------------------
+				// Exporting compressed textures and meshes (KTX2 / Draco) (TODO: Meshopt)
```

Comment:
Draco compression won't leave any trace in the resulting three.js objects, and Meshopt requires only KHR_mesh_quantization (which is tested for export above). So it *might* just be KTX2 that is a concern here, perhaps we could use the existing coffeemat.glb?

Reply:
I believe when I originally started the PR Draco compression did indeed not work for roundtrips since it could still end up as int8 or so, which wasn't supported in the exporter. May not be the case anymore!

Reply:
I just tried using the existing one, but that actually doesn't export cleanly right now, with an error I don't understand - the readback canvas seems to have the right size but comes out as 0x0:

![20230524-023523_chrome](https://github.com/mrdoob/three.js/assets/2693840/73d9f817-53b4-4280-86ba-7dea54c38172)

I debugged a bit and am not really sure what's going on there...